ESLint/lombiq-base.js

```diff
@@ -0,0 +1,160 @@
+module.exports = {
```

Comment:
Ebben meg az eslint-base-ben rengeteg hibát jelez.

Reply:
Ezek ugye a ruleok és itt így kell, hogy kinézzen, de ez nem a linter rulejait követi

Reply:
Igen, de ez így nagyon zavaró. Az is OK teljesen, ha itt minden linter rule-t deaktíválsz.

Reply:
Ezt azóta megcsináltam

Reply:
Már jó, de minden disable-t commentelj, hogy mi értelme van, mert különben egy idő után nem fogjuk érteni.

Reply:
Ez kimaradt.